BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -115,5 +115,53 @@
     </p>
   </span>
 
+  <% bourreau = Bourreau.find(@tool_config.bourreau_id) %>
```

Comment:
Also, it's better to use the Rails association methods than to do a find() explicitely:

```
   bourreau = @tool_config.bourreau
```